.eslintrc

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Can you rename this file to `.eslintrc.json` as it's current naming is deprecated per http://eslint.org/docs/user-guide/configuring#configuration-file-formats